src/main/java/user_use_case/interactors/UpgradeUserResponseModel.java

```diff
@@ -0,0 +1,18 @@
+package user_use_case.interactors;
+
+import entities.OwnerUser;
+
+public class UpgradeUserResponseModel {
+    OwnerUser ownerUser;
+
+    Boolean status;
+
+    public UpgradeUserResponseModel(OwnerUser ownerUser, Boolean status) {
+        this.status = status;
+        this.ownerUser = ownerUser;
+    }
```

Comment:
you wouldnt need to pass through owneruser here, you just need to call Iframe.refresh or back depending on where u want to go after editing